guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2925 @@
+{
```

Comment:
<p>"The <code>get_countries()</code> method can be used to query the countries for which there is GeoEnrichment data. You get back a list of <code>Country</code> object. You can query the properties of each <code>Country</code> object."</p><ul><li>Suggest to combine the three sentences together, e.g. </li><li>The <code>get_countries()</code> method can be used to query the countries for which GeoEnrichment data is available, and it returns a list of <code>Country</code>  objects with which you can further query for properties.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>